### PR TITLE
Transition logic in testJobTemplate into jck.mk

### LINKED ISSUE #1754: Enhancement: Transition JCK logic in testJobTemplate into jck.mk
Currently, there's some jck code in testJobTemplate.

```
if (GROUP == "jck" && JDK_IMPL == "openj9") {
	EXTRA_OPTIONS = "-Xfuture"
	if (JDK_VERSION != "8") {
		EXTRA_OPTIONS += " --enable-preview"
	}
	if (ARCH_OS == "s390x_zos") {
		// zOS has Attach API disabled by default
		EXTRA_OPTIONS += " -Dcom.ibm.tools.attach.enable=yes"
	}
}
```

This issue covers moving that into jck.mk, and any related changes.

As mentioned in https://github.com/AdoptOpenJDK/openjdk-tests/pull/1753 

**📋 Step by Step**

To solve this issue and contribute a fix you should check the following step-by-step list. A more detailed documentation of the workflow can be found [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).

- [ ] Claim this issue: Comment below.
- [ ]  Fork the [openjdk-test](https://github.com/AdoptOpenJDK/openjdk-tests) in github by simply clicking the 'fork' button.
- [ ]  Check out the forked repository
- [ ]  Using any kind of git client (command line, Git plugin on Eclipse IDE, etc) create a feature branch off of your fork for the issue. We do not have any naming definition for branches.
- [ ] At this point, you'd have a local clone of openjdk-test](https://github.com/AdoptOpenJDK/openjdk-tests). Update the jck.mk file with your change in the local clone. 
- [ ]  Commit your changes.
- [ ]  Start a Pull Request.
- [ ]  Done 👍 Ask in comments for a review :)
- [ ]  If the reviewers find some missing peaces or a problem, s/he will start a discussion with you and describe the next steps how the problem can be solved.
- [ ]  You did it 🎉 We will merge the fix in the master branch.
- [ ]  Thanks, thanks, thanks for being part of this project as an open source contributor ❤️

**🤔❓ Questions**

If you have any questions just ask us directly in this issue by adding a comment. You can join our community chat at [Slack](https://adoptopenjdk.net/slack.html). Next to this you can find a general manual about open source contributions [here](https://github.com/firstcontributions/first-contributions/blob/master/README.md).

### PR DESCRIPTION
**Pull request by keerthi-kamath**
Fixes #1754 
Transition JCK logic in testJobTemplate into jck.mk

### CODE REVIEW

File: jck/jck.mk [hunk 1]
```diff
@@ -60,7 +60,17 @@ endif
...
+ String separatorHelpTextStyleCss = ' border: none; margin-top: 0; margin-bottom: 0 '
+ARCH_OS_LIST.each { ARCH_OS ->
+	JDK_VERSIONS.each { JDK_VERSION ->
+		LEVELS.each { LEVEL ->
+			GROUPS.each { GROUP ->
+				def ACTUAL_TEST_JOB_NAME = TEST_JOB_NAME ? TEST_JOB_NAME : "Test_openjdk${JDK_VERSION}_${JDK_IMPL_SN}_${LEVEL}.${GROUP}_${ARCH_OS}${SUFFIX}"
+				def ACTUAL_TARGET = TARGET ? TARGET : "${LEVEL}.${GROUP}"
+				def ACTUAL_BUILD_LIST = BUILD_LIST ? BUILD_LIST : GROUP
+
+
```
Comment (llxia) [inline jck/jck.mk:73]: Not really. I posted the code in https://github.com/adoptium/aqa-tests/issues/1754#issuecomment-932460305. We need to add the code here.

Comment (keerthi-kamath) [reply jck/jck.mk:73]: Oh okay

**Review by llxia — APPROVED** [review]

**Comment by llxia** [comment]
@Mesbah-Alam review reminder. Thanks

File: jck/jck.mk [hunk 2]
```diff
@@ -60,7 +60,13 @@ endif
...
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
- OTHER_OPTS=-Xtrace:maximal=all{level2}
+ OTHER_OPTS=" -Xtrace:maximal=all{level2} -Xfuture "
+ ifeq ($(OS),OS/390)
+     OTHER_OPTS +=" -Dcom.ibm.tools.attach.enable=yes"
+  endif
+  ifneq (8, $(JDK_VERSION))
+    OTHER_OPTS += " --enable-preview"
+  else
```
Comment (Mesbah-Alam) [inline jck/jck.mk:69]: On line 69, instead of 'else', it should be 'endif'.

File: jck/jck.mk [hunk 3]
```diff
@@ -60,7 +60,13 @@ endif
 OTHER_OPTS=
 # if JDK_IMPL is openj9 or ibm
 ifneq ($(filter openj9 ibm, $(JDK_IMPL)),)
- OTHER_OPTS=-Xtrace:maximal=all{level2}
+ OTHER_OPTS=" -Xtrace:maximal=all{level2} -Xfuture "
+ ifeq ($(OS),OS/390)
```
Comment (Mesbah-Alam) [inline jck/jck.mk:64]: On line 64, please fix indentation. The line is missing a space at the beginning.

**Comment by llxia** [comment]
@keerthi-kamath could you please update the PR? We would like to get this PR merged in. Thanks

**Comment by llxia** [comment]
Close this issue as it is obsolete.

**Pull request closed without merging**